Backpack-SwiftUI/Calendar/Classes/CalendarMonthAccessoryAction.swift

```diff
@@ -22,13 +22,14 @@ import SwiftUI
 ///     be performed when a user interacts with an accessory in the calendar.
 public struct CalendarMonthAccessoryAction {
```

Comment:
I think we may need to introduce an enum here, to encapsulate the behaviour. Using this action for for range selection is stretching this api to a point where its not clear, or not as clear as before. I suggest introducing something like so:
```swift
public struct CalendarMonthAccessoryAction {
  let title: String
  let action: Action
  
  public enum Action {
     case custom((Date) -> Void)
     case wholeMonthSelection(ClosedRange<Date> -> Void)
  }
}
```
or something like that, so that when clients use this, its more clear how it will behave and its more explicit